scripts/wfl/twisttcap/find_running_wfl_workloads.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import json
+import requests
+import google.auth as googleauth
+from firecloud import api as fapi
+from time import sleep
+from pprint import pprint
+
+
+def get_access_token():
```

Comment:
Suggestion to have these two functions get_access_token and get_headers in their own _file.py_ and not repeated in the two files.

Reply:
oh great call, definitely want to do this. thanks!